.github/workflows/update_application_statuses.yml

```diff
@@ -0,0 +1,82 @@
+name: Schedule Update Application Statuses
```

Comment:
@slawosz @BroiSatse, I moved out the cron job into a separate container.

Reply:
Not quite sure if I need to do anything else to add this workflow to Github actions and make it a part of the deployment process?